### ISSUE TITLE
The plain word nspages throws a 403 error.

### ISSUE THREAD

**Issue by davidqxo** [gturri/nspages#110]
Nspages version 2020-11-25 (and the previous one). I have a page with a list of plain text names of extensions to consider using. The list of names, one per line, is enclosed in `<text></text>` tags. With the unadorned word *nspages* in the list, a Preview or Save throws a 403 Forbidden error.

If I move the word nspages outside the text tags. No problem.

**Comment by gturri** [comment]
Thank you for this bug report.

Unfortunately I'm not able to reproduce this behavior so I'm going to need more details.
- Could you give me a content of page which triggers this behavior? (ideally a small example)
-  I'm not sure I understand what the `<text>` tags stand for: IIUC it is not part of Dokuwiki standard syntax. Is it provided by a plugin?
- I don't clearly understand if your are using this nspages plugin on the page you're trying to save. Is there some `<nspages>` tag on it?

**Comment by davidqxo** [comment]
Dear Guillaume,

Here is a sample of the Source text that shows the problem. If you take the
word *nspages* below, and move that up to within the "text" tags above at
the beginning of a line (in the ways I tried it), then a Preview or Save
throws a 403 Prohibited error. More details below the sample:

==== plugins ====
=== Recommended ===
<text>
BatchEdit (huge timesaver)
CAPTCHA (anti-spam)
configuration file manager
CSV
Dw2Pdf (export pages as PDFs)
Move
ODT (export to open documents)
orphanswanted
</text>


nspages  (Throws 403 error if the word "nspages" is found within text tags.)


  - I have the nspages plugin installed, but I am not trying to invoke it.
  - Link to the plaintext plugin: https://www.dokuwiki.org/plugin:plaintext

**Comment by gturri** [comment]
Thanks for this information.
Unfortunately I'm still not able to reproduce this error (even after I
installed the plaintext plugin).

I suspect it might come from another source. To check this, could you
please:
- switch to the default template
- uninstall or deactivate all your other plugins
- uninstall or deactivate all templates
and check if the issue is still there.

If not, then you could reactivate all the plugins and templates one by one
in order to see at which point it breaks.

Le ven. 27 nov. 2020 à 19:26, David Phillips <notifications@github.com> a
écrit :

**Comment by davidqxo** [comment]
Thank you for looking into this.

For information, I am using the default template and have loaded no others.

To prepare to work through the debugging steps you suggested, I tried
creating a test page. Unfortunately, I tested the wrong case first--with
plaintest disabled. Perhaps it was the Reinstall/Enable that fixed things
so that I couldn't recreate the problem.

Again, thank you, for your development work and your support.
  -David
